wsuwp-admin.php

```diff
@@ -600,6 +600,7 @@ public function all_plugins( $plugins ) {
 			'footballweekends.wsu.edu/',
 			'stage.native.wsu.edu',
 			'native.wsu.edu',
+			'research.wsu.edu'
```

Comment:
@clintyoung This is good. Next time add a trailing comma - that will make for a cleaner diff when we add another domain.